nsxt/utils_test.go

```diff
@@ -86,6 +87,14 @@ func getMacPoolName() string {
 	return name
 }
 
+func getIpPoolName() string {
+	name := os.Getenv("NSXT_TEST_IP_POOL")
+	if name == "" {
+		name = ipPoolDefaultName
```

Comment:
I don't think nsx provides  default IP pool. We need to skip the  test if pool is not provided for the test in the environment (otherwise test fails)

Reply:
Test is now skipped if NSXT_TEST_IP_POOL is not set